livekit_analytics.proto

```diff
@@ -38,6 +51,8 @@ message AnalyticsStat {
   float connection_score = 16;
   string track_id = 17;
   uint32 rtt = 18;
```

Comment:
This is not used and it was at the end. So, I was tempted to take it out. But, left it in there just in case it breaks other stuff out there.

Reply:
With reference to the `delay` comment above, actually this is more useful when available. `delay` is not. Will look at populating this when available.